docs/commands/stat.asciidoc

```diff
@@ -5,8 +5,8 @@
 The +stat+ command displays a statistic calculated from the features in a map.
 
 * +input+  -  An OSM data input
-* +visitor+ - The stat visitor must be the single class name of a Hootenanny visitor that inherits from ConstElementVisitor and 
-              supports the SingleStatistic interface.  Use the +info --operators+ command to obtain a list of available Hootenanny visitors.
+* +visitor+ - The stat visitor must be the single class name of a Hootenanny visitor that inherits from ElementVisitor and supports the 
+              SingleStatistic interface.  Use the +info --operators+ command to obtain a list of available Hootenanny visitors.
 
```

Comment:
stat command can now take in either a ElementVisitor or ConstElementVisitor for filtering